test/integration/install/spec.rb

```diff
@@ -0,0 +1,46 @@
+# Encoding: utf-8
```

Comment:
Line after Encoding inconsistent in most of the files. There should probably be a new line here.